src/include/migraphx/config.hpp

```diff
@@ -24,10 +24,17 @@
 #ifndef MIGRAPHX_GUARD_CONFIG_HPP
 #define MIGRAPHX_GUARD_CONFIG_HPP
 
+#include <migraphx/version.h>
 namespace migraphx {
 
 #if !defined(MIGRAPHX_USE_CLANG_TIDY) && !defined(DOXYGEN)
-#define MIGRAPHX_INLINE_NS version_1
+
+#define MIGRAPHX_CONCAT_1(x, y) x##y
+#define MIGRAPHX_CONCAT(x, y) MIGRAPHX_CONCAT_1(x, y)
+
+#define MIGRAPHX_VERSION MIGRAPHX_CONCAT(MIGRAPHX_VERSION_MAJOR, MIGRAPHX_VERSION_MINOR)
+
+#define MIGRAPHX_INLINE_NS MIGRAPHX_CONCAT(VERSION, MIGRAPHX_VERSION)
```

Comment:
There should be a `BUILD_DEV` flag in cmake to disable this.

Reply:
To improve ccache in the dev environment.

Reply:
I'll keep this and adding `tweak` and `patch` for another PR. For now, If it's okay, i'll not make any further changes in `config.hpp`.

Reply:
It would be better to remove this for this PR otherwise everything needs to be rebuilt, and then rebuilt again when we update this with `BUILD_DEV` flag.

Reply:
Okay. https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/issues/1627

Reply:
Removed.